artifacts/linting_rules/.spectral.yml

```diff
@@ -1,6 +1,6 @@
 # CAMARA Project - linting ruleset - documentation avaialable here:
 # https://github.com/camaraproject/Commonalities/blob/main/documentation/Linting-rules.md
-# 31.01.2024 - initial version
```

Comment:
If change comments are added with dates ... wouldn't it then make sense to keep the previous lines and create a kind of changelog by that?

Reply:
Yes, it is possible way. I assumed that the changes can be tracked using file history in the Github, and files are marked with timestamp to easily verify if linting configuration in subproject is up to date with Commonalities artifacts.
Since it is the first correction of the linting ruleset we can start using the changelog style versioning if it doesn't clutter the files.